src/elin_bjoernson_ex/ex02/bubble_sort.py

```diff
@@ -0,0 +1,22 @@
+# -*- coding: utf-8 -*-
+
+
+def bubble_sort(numbers):
+    new_list = list(numbers)
+    for _ in new_list:
+        for idx, num in enumerate(new_list):
+            if idx == len(new_list)-1:  # if last element, do nothing
+                pass
+            elif num > new_list[idx+1]:  # exchange if higher value than next
+                new_list.insert(idx+1, new_list.pop(idx))
```

Comment:
`new_list[idx] , new_list[idx+1] = new_list[idx+1] , new_list[id]` This's a better way to swap elements in a list.